src/setuptools_betterproto/_config.py

```diff
@@ -99,3 +100,13 @@ def from_strings(
             include_paths=[p.strip() for p in include_paths.split(",")],
             out_path=out_path,
         )
+
+    @property
+    def expanded_proto_files(self) -> list[str]:
+        """The files in the `proto_path` expanded according to the configured glob."""
+        proto_path = pathlib.Path(self.proto_path)
+        return [str(proto_file) for proto_file in proto_path.rglob(self.proto_glob)]
+        return [
+            str(proto_file.relative_to(proto_path))
+            for proto_file in proto_path.rglob(self.proto_glob)
+        ]
```

Comment:
there is double return here but it seems it was fixed in another commit

Reply:
it was fixed in the commit "Distribute protobuf files automatically"

Reply:
Ah, thanks, there was a lot of rebasing and cherry-picking of diff lines, I guess I messed it up at some point, will fix!